run/GCHP/createRunDir.sh

```diff
@@ -786,17 +787,19 @@ else
 	fi
     fi
     if [[ ${sim_extra_option} =~ "TOMAS" ]]; then
+	RUNDIR_VARS+="RUNDIR_DUSTL23M_EXT='off'\n"
 	RUNDIR_VARS+="RUNDIR_TOMAS_DUSTDEAD='on '\n"
 	RUNDIR_VARS+="RUNDIR_OFFLINE_DUST='false'\n"
     else
+	RUNDIR_VARS+="RUNDIR_DUSTL23M_EXT='off'\n"
 	RUNDIR_VARS+="RUNDIR_TOMAS_DUSTDEAD='off'\n"
 	RUNDIR_VARS+="RUNDIR_OFFLINE_DUST='true '\n" 
     fi
-    RUNDIR_VARS+="RUNDIR_DUSTDEAD_EXT='off'\n"
     RUNDIR_VARS+="RUNDIR_MEGAN_EXT='off'\n"
     RUNDIR_VARS+="RUNDIR_SOILNOX_EXT='off'\n"
     RUNDIR_VARS+="RUNDIR_OFFLINE_BIOVOC='true '\n"
     RUNDIR_VARS+="RUNDIR_OFFLINE_SOILNOX='true '\n"
+    RUNDIR_VARS+="RUNDIR_DUSTDEAD_EXT='off'\n"       # No longer used
```

Comment:
Should this be deleted?

Reply:
@lizziel: Now fixed in commit a4fdcdcba.